README.md

```diff
@@ -608,8 +608,31 @@ After that you can use the extension in your code with full intellisense.
 await tatumSdk.extension(HelloWorldExtension).sayHello()
 ````
 
+### Wallet Providers
+
+Tatum SDK supports wallet provider extensions for various wallets. You can use them to integrate your application with the wallet of your choice.
+
+```typescript
+import { TatumSDK, Ethereum, Network, ApiVersion } from '@tatumio/tatum'
+import { HelloWorldExtension } from "@tatumio/wallet-provider-demo"
```

Comment:
I would suggest here to use some real world example - Metamask. Because it will be easier for everyone to just copy paste, instead of decoding what it is wallet-provider-demo

Reply:
yes, was going to do that but MetaMask is built in so you don't have to register it - it would be even more confusing. We need to change this to something else once something else is avaliable.

Reply:
Ok. Then for me everything is good.